src/main/scala/ai/eto/rikai/RikaiOptions.scala

```diff
@@ -21,4 +21,10 @@ private[rikai] class RikaiOptions(parameters: Map[String, String]) {
   /** Base path for the feature dataset
     */
   val path: String = parameters.getOrElse("path", "")
+
+  val defaultBlockSize: Int = 32 * 1024 * 1024
+
+  /** Parquet block size. */
+  val blockSize: Int =
+    parameters.getOrElse("rikai.block.size", s"${blockSize}").toInt
```

Comment:
Should the format str be defaultBlockSize?

Reply:
Ah, was made a mistake in the latest cleanup

Reply:
how come the tests passed tho?